web/src/components/schedule/subcomponents/thursday_schedule/thursday_schedule.js

```diff
@@ -0,0 +1,104 @@
+import { Box, Grid, Stack, Typography } from "@mui/material";
+import ScheduleTimeline from "../schedule_timeline/schedule_timeline.js";
+import thursdayScheduleStyles from "./thursday_schedule.styles.js";
+import { useTheme } from "@emotion/react";
+import messages from "./messages.js";
+
+function ThursdaySchedule() {
+  const theme = useTheme();
+  return (
+    <Box sx={thursdayScheduleStyles.section}>
+      <Stack>
+        <Typography variant="h3">{messages.title}</Typography>
+        <Typography variant="h5" color={theme.palette.text.subtitle}>
+          {messages.subtitle}
+        </Typography>
+        <ScheduleTimeline />
+
+        {/* Headers */}
+        <Grid container spacing={2} columns={{ sm: 4, md: 12 }}>
```

Comment:
Would be good if we can extract this into a re-usable component too

Reply:
marking unresolved

Reply:
Moved into reusable components